### PR TITLE
fix : align progress bar in signup fragment

### LINKED ISSUE #647: Align progress bar position.
**Describe the bug**
misaligned progress bar in singup fragment

**Screenshots**
whatsapp image 2018-10-16 at 8 33 12 am

**Smartphone Info:**

|               | |
|---------------|-|
|Device         |Oneplus 5|
|Android Version|    8.1|

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by shikhart98**
Fixes #647 

I have a doubt ! Initially in Utils I have created a similar function and implemented the layout as shown. Should I do the same for every other layouts where this orientation is required?

Screenshots for the change:
whatsapp image 2018-10-29 at 8 38 10 am

### CODE REVIEW

**Comment by iamareebjamal** [comment]
That's too much changes for a simple fix

File: app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt [hunk 1]
```diff
@@ -77,8 +77,11 @@ class SignUpFragment : Fragment() {
                 }
             }
 
-            override fun beforeTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) { /*Implement here*/ }
-            override fun onTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) { /*Implement here*/ }
+            override fun beforeTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) { /*Implement here*/
+            }
+
```
Comment (simarsingh24) [inline app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt]: these are unnecessary changes

File: app/src/main/java/org/fossasia/openevent/general/utils/Utils.kt [hunk 2]
```diff
@@ -32,6 +33,11 @@ object Utils {
         progressBar.visibility = if (show) View.VISIBLE else View.GONE
     }
 
+    fun showProgressBar(progressBar: ProgressBar, background:FrameLayout, show: Boolean) {
+        progressBar.visibility = if (show) View.VISIBLE else View.GONE
+        background.visibility = if(show) View.VISIBLE else View.GONE
```
Comment (simarsingh24) [inline app/src/main/java/org/fossasia/openevent/general/utils/Utils.kt]: two checks are not needed can be done using one
